### PR TITLE
bookmarks/saving a recipe

### PR DESCRIPTION
**Pull request by bellehlim**
users can now save recipes via icon on recipe card and it will save to their bookmarks page. state of icon + number of saves is also active and works immediately. only effect not immediately visible is if you remove a recipe from your bookmarks, you have to refresh to see it disappear.

jenky approach but it works :)))

### CODE REVIEW

File: src/pages/bookmarksPage.tsx [hunk 1]
```diff
@@ -1,11 +1,29 @@
...
+      }
+    };
+    fetchPosts();
+  }, [currentUser]);
+  
+  return (
+    <div>
+      <h2> My Bookmarks</h2>
+      {currentUser ? <RecipeList recipesArray={postBody}/>
+          : <p>See your bookmarks here when you log in!</p>}
```
Comment (surbhi-gulati) [inline src/pages/bookmarksPage.tsx:24]: add login button

File: src/reducers/bookmarks-reducer.ts [hunk 2]
```diff
@@ -1,97 +1,97 @@
...
-          state.bookmarksByUserId[deletedBookmark.userId] = state.bookmarksByUserId[
-            deletedBookmark.userId
-          ].filter((bookmark) => bookmark.id !== deletedBookmark.id);
-        }
-      });
-  },
-});
-
-export default bookmarksSlice.reducer;
+// import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
```
Comment (surbhi-gulati) [inline src/reducers/bookmarks-reducer.ts]: nit: should we leave the 2 reducers uncommented even if not in use?

**Review by surbhi-gulati — APPROVED** [review]

File: src/reducers/bookmarks-reducer.ts (hunk 2 above)
Comment (bellehlim) [reply src/reducers/bookmarks-reducer.ts]: oops forgot to straight up delete them

File: src/pages/bookmarksPage.tsx (hunk 1 above)
Comment (bellehlim) [reply src/pages/bookmarksPage.tsx:24]: we already technically have the login button at the top, here's how it looks if we have both?
image

**Pull request merged**